package/libs/libbsd/Makefile

```diff
@@ -27,30 +26,17 @@ define Package/libbsd/description
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) \
-		$(1)/lib \
-		$(1)/usr/include
-
-	$(CP) \
-		$(PKG_INSTALL_DIR)/lib/libbsd.so* \
-		$(1)/lib/
-
-	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/* \
-		$(1)/usr/include/
-
-	( cd $(1)/lib ; $(LN) libbsd.so.$(PKG_VERSION) libbsd.so )
+	$(INSTALL_DIR) $(1)/usr/lib
```

Comment:
Can all 3 directories be created on the same line?

Reply:
Dunno? :-)
You like ```$(INSTALL_DIR) $(1)/usr/lib $(INSTALL_DIR) $(1)/usr/lib/pkgconfig $(INSTALL_DIR) $(1)/usr/include``` ?
Wouldn't that get too long?

Reply:
Should be fine

Reply:
I honestly think that's harder to read but if the preferred way I can change it.
If we should keep the style it needs to be broken down into two lines which doesn't save much in the end?

Reply:
It’s about seeing up front all the directories that need to be created without having to search them out.